ui/app/pages/swaps/select-quote-popover/index.scss

```diff
@@ -39,54 +39,43 @@
 
   &__sort-list {
     display: flex;
-    flex-flow: column;
+    flex-direction: column;
+    align-items: center;
+  }
+
+  &__column-headers,
+  &__row {
+    display: flex;
```

Comment:
A few rules common to both `column-headers` and `row` were consolidated here.

I didn't move all of the common rules here - just the ones that seemed unlikely to change, e.g. the layout-related ones.